docs/content/guides/6.multistore/3.patterns/3.data/1.different-config-same-integration.md

```diff
@@ -6,3 +6,211 @@ navigation:
+3. Each store will have its own configuration with values specific to its needs
+
+### Implementation Example
+
+For our SAP Commerce Cloud example with B2C and B2B stores:
+
+```ts
+// B2C store
+// /apps/stores/b2c/storefront-middleware/integrations/sapcc/config.ts
+
+export const config = {
+  configuration: {
+    api: {
+      baseSiteId: 'b2c-site', // store specific value
+      catalogId: 'b2cProductsCatalog', // store specific value
+      // ...
+    },
+  }
+}
+```
+
+```ts
+// B2B store
+// /apps/stores/b2b/storefront-middleware/integrations/sapcc/config.ts
+
+export const config = {
+  configuration: {
+    api: {
+      baseSiteId: 'b2b-site', // store specific value
+      catalogId: 'b2bProductsCatalog', // store specific value
+      // ...
+    },
+  }
+}
+```
+
+After this operation, the file structure will look like this:
+
+```bash
+apps/
+├── storefront-middleware
+|  ├── integrations
+|  |  └── sapcc
+|  |     └── config.ts # base config will be overridden by stores config
+└── stores
+   ├── b2c
+   |  ├── storefront-middleware
+   |  |  ├── integrations
+   |  |  |  └── sapcc
+   |  |  |     └── config.ts # new file for b2c store 
+   └── b2b
+      └── storefront-middleware
+         └── integrations
+            └── sapcc
+               └── config.ts # new file for b2b store 
+```
+
+:::tip
+This approach leverages the [file-based inheritance](/guides/multistore/tooling-and-concepts/file-based-inheritance) system in Alokai, allowing you to override only what's different while inheriting everything else.
+:::
+
+## Best Practices
+
+### 1. Use Environment Variables
+
+Instead of creating separate configuration files for each store, leverage **environment variables** where possible. Different environment settings can dynamically configure the middleware or storefront application for each store.
+
+This approach helps avoid duplicated files and makes your solution more modular, clean, and easy to maintain.
+
+Let's improve our previous configuration example to use environment variables:
+
+```ts
+// base source code, not nested in stores
+// /apps/storefront-middleware/integrations/sapcc/config.ts
+
+const { BASE_SITE_ID, PRODUCT_CATALOG_ID } = process.env;
+
+export const config = {
+  configuration: {
+    api: {
+      baseSiteId: BASE_SITE_ID, // dynamic value
+      catalogId: PRODUCT_CATALOG_ID, // dynamic value
+      // ...
+    },
+  }
+}
+```
+
+For local development, use `.env` files specific to each store:
+
+```plaintext
+# b2c store
+# /apps/stores/b2c/storefront-middleware/.env
+BASE_SITE_ID="b2c-site"
+PRODUCT_CATALOG_ID="b2cProductsCatalog"
+
+# b2b store
+# /apps/stores/b2b/storefront-middleware/.env
+BASE_SITE_ID="b2b-site"
+PRODUCT_CATALOG_ID="b2bProductsCatalog"
+```
+
+:::warning
+Ensure that `.env` files are versioned appropriately. To avoid accidentally committing sensitive configurations, use `.env.example` to provide a template and add `.env` files to `.gitignore`.
```

Comment:
I think that `.env` files are already in `.gitignore` this sentence suggest that the developer has to ignore them manually.

Reply:
That is correct yet this is just a reminder, a double check. If this is misleading, i'll remove it.

Reply:
I've updated this info.